### PR TITLE
Added logging to api-updates-subscribe element, added siteimprove script

### PR DESCRIPTION
**Pull request by e-kr**
Added logging for the api-updates-subscribe custom element when user opens modal and when clicking Subscribe. It only logs when clicking subscribe if inputs are valid. Also added the script from design system.

Check if you agree with the logging values :)

### CODE REVIEW

**Comment by WebDevBring** [comment]
[trello.com/c/Cd0BQCmA](https://trello.com/c/Cd0BQCmA)

File: js/webcomponent/api-updates-subscribe/main.js [hunk 1]
```diff
@@ -141,6 +150,15 @@ class ApiSubscribe extends HTMLElement {
           "Invalid email format!"
         )
       }
+
+      // Log to SiteImprove when clicking Subscribe
+      if(typeof SiteImprove !== 'undefined' && (apiAreaRadios[0].checked || someSelected)){
+        SiteImprove.registerEvent({
+          category: 'Task',
+          action: 'Subscribe to API updates',
+          label: 'Subscribe to API updates'
```
Comment (Igor-J86) [inline js/webcomponent/api-updates-subscribe/main.js:159]: A suggestion after discussions:

```suggestion
          label: 'Confirm subscription'
```

File: js/webcomponent/api-updates-subscribe/main.js [hunk 2]
```diff
@@ -49,6 +49,15 @@ class ApiSubscribe extends HTMLElement {
     if (closeModalBtn) {
       closeModalBtn.focus()
     }
+
+    // Log to SiteImprove when opening the modal
+    if(typeof SiteImprove !== 'undefined'){
+      SiteImprove.registerEvent({
+        category: 'Flow steps',
```
Comment (Igor-J86) [inline js/webcomponent/api-updates-subscribe/main.js:56]: We will update the documentation in MDS so we use singular form on the predefined categories:

```suggestion
        category: 'Flow step',
```

File: js/webcomponent/api-updates-subscribe/main.js (hunk 1 above)
Comment (e-kr) [reply js/webcomponent/api-updates-subscribe/main.js:159]: What about "Confirm email subscription" to also include email? Or is it not necessary 🤔

Comment (Igor-J86) [reply js/webcomponent/api-updates-subscribe/main.js:159]: I think we‘ve concluded that it is sufficient with "Confirm subscription". 😄

**Review by Igor-J86 — APPROVED** [review]
Nice! 🚀

**Pull request merged**